infra/token-transfer-server/src/controllers/transfer.js

```diff
@@ -0,0 +1,92 @@
+const express = require('express')
+const router = express.Router()
+const { check, validationResult } = require('express-validator')
+const moment = require('moment')
+
+const { createProvider } = require('@origin/token/src/config')
+
+const logger = require('../logger')
+const { Grant, Transfer } = require('../../src/models')
+const { ensureLoggedIn } = require('../lib/login')
+const {
+  asyncMiddleware,
+  isEthereumAddress,
+  getUnlockDate
+} = require('../utils')
+const { unlockDate, networkId } = require('../config')
+const { enqueueTransfer } = require('../lib/transfer')
+
+createProvider(networkId) // Ensure web3 credentials are set up
```

Comment:
Remove - this does not seem to be used anywhere ?
And anyway I refactored the token package so this method no longer exist... :)